### DISCUSSION TITLE
Show total Size of Plots assigned to Pool

### DISCUSSION THREAD

**Discussion by Muffexx** [Chia-Network/chia-blockchain#8080] [Ideas]
Hello there,

currently the Pool Overview does only show the "Number of Plots" assigned to any pool. However when using Plots of different sizes this information is not as meaningful as desired.
My suggestion would be to show the total capacity as a separate line or to show the total capacity when hovering over the number of plots.

regards

**Comment by BrandtH22** [comment]
@Muffexx , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
